test/chemistry/test_uccsd_hartree_fock.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
         self.optimizer = SLSQP(maxiter=100)
         initial_state = HartreeFock(
             fermionic_transformation.molecule_info['num_orbitals'],
-            fermionic_transformation.molecule_info['num_particles'],
+            tuple(fermionic_transformation.molecule_info['num_particles']),
```

Comment:
Should we not change this at the source to emit a tuple if list is going to be deprecated. Otherwise its wrong out gate.

Reply:
Good point 👍

Reply:
done in 608aade